packages/lpa-questionnaire-web-app/src/config.js

```diff
@@ -1,4 +1,5 @@
 const fifteenMegabytes = 1024 * 1024 * 15;
+const httpPort = Number(process.env.PORT || 3000);
```

Comment:
What's the reason for adding this now? Nothing in the ticket as far as I can see?

Reply:
It's referenced by the server host further on in the page which is required for the pdf service.